docs/deployment/landing_page.rst

```diff
@@ -249,3 +249,77 @@ that user and group file permissions are locked down and that modules or
 gateway interfaces for dynamic scripting languages are not enabled. You
 don't want any unnecessary code or services running as this increases
 the attack surface.
+
+Landing page content suggestions
+--------------------------------
+
+The content below presents sample text for the SecureDrop component of a news 
+organization’s tips page. It does not account for any specific legal 
+or organizational needs, but should provide guidance for any outlet getting 
+started on crafting landing page language. Any tweaks to the sample content 
+should be left to the legal and editorial discretion of the individual outlet, 
+and should be viewed as essential to upholding source protection and transparency.
+
+**What is SecureDrop?**
+
+SecureDrop is an anonymity tool for journalists and whistleblowers. As a source, 
+you can use our SecureDrop installation to anonymously submit documents to our 
+organization. Our journalists use SecureDrop to receive source materials and 
+securely communicate with anonymous contacts.
```

Comment:
with anonymous contacts => with you ?

Reply:
I prefer "anonymous contacts", since not 100% of people who view the landing page are going to submit information, whether via SecureDrop or otherwise. So the third-person rather than the second-person makes the most sense.

Reply:
I like it as-is, to establish that the motivation is to host multiple relationships with sources in one "inbox".